databus/src/main/java/com/bazaarvoice/emodb/databus/db/cql/CqlSubscriptionDAO.java

```diff
@@ -123,9 +114,9 @@ public OwnedSubscription getSubscription(String subscription) {
     private OwnedSubscription rowToOwnedSubscription(Row row) {
         String name = row.getString(0);
         Map<?, ?> json = JsonHelper.fromJson(row.getString(1), Map.class);
-        Condition tableFilter = Conditions.fromString((String) Objects.requireNonNull(json.get("filter"), "filter"));
-        Date expiresAt = new Date(((Number) Objects.requireNonNull(json.get("expiresAt"), "expiresAt")).longValue());
-        Duration eventTtl = Duration.ofSeconds(((Number) Objects.requireNonNull(json.get("eventTtl"), "eventTtl")).intValue());
+        Condition tableFilter = Conditions.fromString((String) checkNotNull(json.get("filter"), "filter"));
+        Date expiresAt = new Date(((Number) checkNotNull(json.get("expiresAt"), "expiresAt")).longValue());
+        Duration eventTtl = Duration.ofSeconds(((Number) checkNotNull(json.get("eventTtl"), "eventTtl")).intValue());
         // TODO:  Once API keys are fully integrated enforce non-null
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1135)